### ISSUE TITLE
UI improvements for Progress

### ISSUE THREAD

**Issue by Strajk** [kiwicom/stranger-strings#57]
Why?

* Most important info is what **is untranslated**, not what **is translated**, but right now, majority of space is used to show the latter.

Proposed

![image](https://user-images.githubusercontent.com/697301/64018106-c6273000-cb2b-11e9-833b-09d26d9390fb.png)

**Comment by D1LL1G4F** [comment]
I agree that the most important info is untranslated. However, for me, the proposed change looks more like raw data than a helpful overview of the key. I am also not sure if it is even important **how many** important languages are untranslated in comparison to **whenever there is** some untranslated important language (maybe completely removing numbers or moving them to some tooltip would be a good idea). Also, for me, the color is contrast enough to easily spot missing important languages in progress bar.
